sssd_test_framework/misc/ssh.py

```diff
@@ -92,3 +94,74 @@ def __enter__(self) -> SSHKillableProcess:
     def __exit__(self, exception_type, exception_value, traceback) -> None:
         self.kill()
         self.process.wait()
+
+
+Param = ParamSpec("Param")
+
+
+def retry_command(
```

Comment:
How is this going to be used? with retry() as a decorator we can do something like 

```
@retry()
def test_run_this_5x():
```

Some examples in the doc string for retry and retry_command will be nice.